src/panels/OriginPanel/OriginPanel.tsx

```diff
@@ -24,6 +25,8 @@ export function OriginPanel() {
   const properties = useAppSelector((state) => state.properties.properties);
   const engineMode = useAppSelector((state) => state.engineMode.mode);
 
+  const featuredNodes = useFeaturedNodes();
+
   const shouldStartInAnchorAim = useAppSelector((state) => {
```

Comment:
Hook order: 
   - useAppSelector
   - useState
   - useContext
   - useRef
   - customHooks (useOpenSpaceApi, etc)
   - derived state
   - useAppDispatch
   - useEffect
   - useMemo, useCallback
   - functions

Reply:
Thanks for the reminder. Maybe we could merge this PR sometime soon...? 

https://github.com/OpenSpace/OpenSpace-WebGui/pull/73/files